app/code/Magento/UrlRewriteGraphQl/Model/Resolver/EntityUrl.php

```diff
@@ -73,6 +74,11 @@ public function resolve(
         $url = $customUrl ?: $url;
         $urlRewrite = $this->findCanonicalUrl($url);
         if ($urlRewrite) {
+            if (!$urlRewrite->getEntityId()) {
```

Comment:
We hide errors here.

Simple example:
1. Create Custom URL Rewrite
Request Path: go-to-admin
Target Path: /admin
Redirect Type: No
Perform query:
query{
  urlResolver(
    url: "go-to-admin"
  ) {
    id
    relative_url
    type
  }
}

For this case expected response in Internal Server Error which means
```
Exception #0 (LogicException): Front controller reached 100 router match iterations
```
but we simply got "No such entity found with matching URL key: go-to-admin"

Reply:
Maybe it would be better to rely on Magento error handling then just check if our response is casting to  `true` or `false`? @naydav?

Reply:
> Exception #0 (LogicException): Front controller reached 100 router match iterations
Looks like it's bug in Magento. expected result: 404 Page not found

Does everything else work?